core/TypeErrorDiagnostics.cc

```diff
@@ -115,4 +116,22 @@ void TypeErrorDiagnostics::insertUntypedTypeArguments(const GlobalState &gs, Err
         }
     }
 }
+
+void TypeErrorDiagnostics::explainUntyped(const GlobalState &gs, ErrorBuilder &e, ErrorClass what,
+                                          const TypeAndOrigins &untyped, Loc originForUninitialized) {
+    e.addErrorSection(untyped.explainGot(gs, originForUninitialized));
+    if (what == core::errors::Infer::UntypedValue) {
+        e.addErrorNote("Support for `{}` is minimal. Consider using `{}` instead.", "typed: strong", "typed: strict");
+    }
+}
+
+void TypeErrorDiagnostics::explainUntyped(const GlobalState &gs, ErrorBuilder &e, ErrorClass what, TypePtr untyped,
+                                          Loc origin, Loc originForUninitialized) {
+    auto untypedTpo = TypeAndOrigins{untyped, origin};
+    e.addErrorSection(untypedTpo.explainGot(gs, originForUninitialized));
+    if (what == core::errors::Infer::UntypedValue) {
+        e.addErrorNote("Support for `{}` is minimal. Consider using `{}` instead.", "typed: strong", "typed: strict");
```

Comment:
Also OOC, what do you think about this?
```suggestion
        e.addErrorNote("Support for `typed: strong` is minimal. Consider using `typed: strict` instead.");
```

I saw this in multiple places and always wondered why it is the way it is

Reply:
The logic that adds colors to things in the command line looks specifically for a raw ``"`{}`"`` substring in a format string:

https://github.com/sorbet/sorbet/blob/38a4701bd236e83786c9abee547638c3766daaf2/core/Error.h#L30

If I were to apply the suggestion as proposed, it would not highlight `typed: strong` in cyan in the terminal, and would instead show backticks.

There are some errors in Sorbet that need to be rewritten to use ``"`{}`"`` but I haven't had time to take a pass at fixing those.